lib/puppetserver/ca/utils/file_system.rb

```diff
@@ -50,6 +50,17 @@ def self.check_for_existing_files(one_or_more_paths)
           errors
         end
 
+        def self.symlink_to_old_cadir(current_cadir)
+          old_cadir = '/etc/puppetlabs/puppet/ssl/ca'
+          new_cadir = '/etc/puppetlabs/puppetserver/ca'
```

Comment:
Should I use the same `running_as_root?` branching that Justin had in his PR? That might make testing possible.. But not sure if we really only wanna do this linking if we're using that^ specific default.

Reply:
imo, having running from source be as similar to running from packages is something we should shoot for.

Reply:
I'm not sure what you mean. Are you saying I should or should not do the same branching as you did?

Reply:
Yeah, I'd prefer we do the branching.